src/mumble/MainWindow.cpp

```diff
@@ -1706,13 +1710,15 @@ void MainWindow::on_qaUserLocalIgnoreTTS_triggered() {
 	p->setLocalIgnoreTTS(ignoredTTS);
 	if (! p->qsHash.isEmpty())
 		g.db->setLocalIgnoredTTS(p->qsHash, ignoredTTS);
+	else
+		msgBox(MainWindow::tr("This change could not be saved permanently and is lost on restart because %1 does not have a certificate.").arg(Log::formatClientUser(p, Log::Target)));
 }
 
 void MainWindow::on_qaUserLocalVolume_triggered() {
 	ClientUser *p = getContextMenuUser();
-	if (!p) {
+	if (!p)
 		return;
-	}
```

Comment:
This has obviously nothing to do with the PR but I saw that it was inconsistent to the other slots and changed.

Reply:
We're actually trying to move away from the non-brace variant, so please don't remove braces ^^

Reply:
> We're actually trying to move away from the non-brace variant, so please don't remove braces ^^

https://wiki.mumble.info/wiki/Coding_Guidelines#if_statements should be updated then

Reply:
Indeed. These guidelines are probably outdated anyways though and will be replaced by guidelines here o GitHub at some point...

Reply:
I put the braces back in and also added them in all other functions that I have changed